task/handler/core/src/main/java/io/knotx/fragments/task/handler/consumer/MetadataConverter.java

```diff
@@ -69,7 +69,7 @@ private void setGraphLogPropertiesFrom(GraphNodeExecutionLog graphLog, NodeExecu
     if (metadataResponse != null) {
       graphLog
           .setResponse(GraphNodeResponseLog.newInstance(metadataResponse.getTransition(),
-              metadataResponse.getInvocations()));
+              metadataResponse.getLog()));
```

Comment:
Should pass `metadataResponse.getErrors()` to factory method?

Reply:
Fixed.